controllers/keyword.go

```diff
@@ -30,6 +31,38 @@ func (c *Keyword) Get() {
 	flash.Store(&c.Controller)
 }
 
+func (c *Keyword) Show() {
+	keyword, err := c.getKeyword()
+	if err != nil {
+		logs.Error("Error when getting keyword: %v", err)
+		c.Redirect("/", http.StatusNotFound)
+		return
+	}
+
+	keywordPresenter := presenters.KeywordPresenter{Keyword: keyword}
+
+	c.Data["Keyword"] = keyword
+	c.Data["KeywordLinks"] = keywordPresenter.KeywordLinks()
```

Comment:
Since `keyword` can be accessed through `keywordPresenter`, how about setting the view data with only the presenter?

```suggestion
	c.Data["keywordPresenter"] = keywordPresenter
```